content/authors/Kelly-Garner/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+# Display name
+title: "Kelly Garner"
+
+# Username (this should match the folder name and the name on publications)
+authors:
+  - "Kelly Garner"
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence) & Organizations/Affiliations
+position: "???"
```

Comment:
@kel-github feel free to update though at the moment I don't think we use it anywhere on the website